src/components/common/YourCompany/YourCompany.vue

```diff
@@ -362,6 +352,7 @@ export default class YourCompany extends Mixins(
   @Getter isBenIaCorrectionFiling!: boolean
   @Getter isFirmCorrectionFiling!: boolean
   @Getter getEntityType!: CorpTypeCd
+  @Getter getCorrectionStartDate!: string
```

Comment:
Looks like this isn't used in this component.

Reply:
I'll remove this